src/commands/ShowDoc.ts

```diff
@@ -0,0 +1,74 @@
+import * as vscode from 'vscode'
+import * as path from "path";
+import * as parser from "../parser"
+import * as extension from "../extension"
+import * as fs from "fs"
+
```

Comment:
We use OOP instead of functional programming.
I suggest you can abstract these functions as methods of a class.